web/src/kubernetes/index.ts

```diff
@@ -0,0 +1,108 @@
+import {
+  KubeConfig,
+  BatchV1Api,
+  CoreV1Api,
+  V1ConfigMap,
+  V1Job} from "@kubernetes/client-node";
+import {Service} from "ts-express-decorators";
+import { Installer } from "../installers";
+import { Templates } from "../util/services/templates";
+import { S3Signer } from "../util/persistence/s3";
+
+@Service()
+export class Kubernetes {
+  private distOrigin: string;
+  private ns: string;
+  private disabled: boolean;
+  private s3Signer: S3Signer;
+  private bucket: string;
+
+  constructor(
+    private readonly templates: Templates,
+  ) {
+    this.distOrigin = `https://${process.env["KURL_BUCKET"]}.s3.amazonaws.com`;
+    this.ns = process.env["KURL_NAMESPACE"] || "kurl";
+    this.disabled = !!process.env["AIRGAP_BUNDLING_DISABLED"];
+    this.s3Signer = new S3Signer();
+    this.bucket = process.env["KURL_BUCKET"] || "kurl-dev";
+  }
+
+  public async runCreateAirgapBundleJob(i: Installer) {
```

Comment:
First create a ConfigMap with three scripts:
1. the bash script to rebundle all the individual packages
2. the rendered install.sh script
3. the rendered join.sh script